[quant] Update qbatch_norm name to qbatch_norm2d

supriyar opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #36552 [quant][graph] Add quantized batch_norm2d_relu to graph mode
* #36549 [quant][graph] Add quantized batch_norm2d support to graph mode
* #36548 [quant][graph] Add valueQuantizable function
* **#36494 [quant] Update qbatch_norm name to qbatch_norm2d**

Summary:
Make name consistent with op. Since we have batch_norm2d and batch_norm3d ops

Test Plan:
python test/quantization/test_quantized.py test_batch_norm2d
Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21008831](https://our.internmc.facebook.com/intern/diff/D21008831)

supriyar committed:
[quant] Update qbatch_norm name to qbatch_norm2d

Summary:
Make name consistent with op. Since we have batch_norm2d and batch_norm3d ops

Test Plan:
python test/quantization/test_quantized.py test_batch_norm2d
Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[quant] Update qbatch_norm name to qbatch_norm2d"


Summary:
Make name consistent with op. Since we have batch_norm2d and batch_norm3d ops

Test Plan:
python test/quantization/test_quantized.py test_batch_norm2d
Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented on aten/src/ATen/native/quantized/cpu/qbatch_norm.cpp:
```diff
@@ -274,7 +274,7 @@ class QBatchNorm3d final : public torch::OperatorKernel {
 
 static auto registry = torch::RegisterOperators().op(
-    "quantized::batch_norm(Tensor qx, "
+    "quantized::batch_norm2d(Tensor qx, "
```
do you need to keep `quantized::batch_norm` for BC?

supriyar replied:
I don't think so. The `aten::quantized_batch_norm` is still preserving bc.
cc @lly-zero-one for new op signature.

lly-zero-one replied:
Can you also change the convert script?

lly-zero-one replied:
NVM, I think I just need to update the model.

supriyar committed:
Update on "[quant] Update qbatch_norm name to qbatch_norm2d"


Summary:
Make name consistent with op. Since we have batch_norm2d and batch_norm3d ops

Test Plan:
python test/quantization/test_quantized.py test_batch_norm2d
Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

supriyar committed:
Update on "[quant] Update qbatch_norm name to qbatch_norm2d"


Summary:
Make name consistent with op. Since we have batch_norm2d and batch_norm3d ops

Test Plan:
python test/quantization/test_quantized.py test_batch_norm2d
Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21008831](https://our.internmc.facebook.com/intern/diff/D21008831)

[ghstack-poisoned]
